simulations/vip-393/ethereum.ts

```diff
@@ -0,0 +1,159 @@
+    await impersonateAccount(USDT_PRIME_CONVERTER);
+    await setBalance(USDT_PRIME_CONVERTER, parseUnits("1000", 18));
+    await impersonateAccount(EIGEN_USER);
+    await setBalance(EIGEN_USER, parseUnits("1000", 18));
+
+    resilientOracle = await ethers.getContractAt(RESILIENT_ORACLE_ABI, ethereum.RESILIENT_ORACLE);
+    poolRegistry = await ethers.getContractAt(POOL_REGISTRY_ABI, ethereum.POOL_REGISTRY);
+    vEIGENContract = await ethers.getContractAt(VTOKEN_ABI, vEIGEN);
+    comptroller = await ethers.getContractAt(COMPTROLLER_ABI, CORE_COMPTROLLER);
+    eigenContract = await ethers.getContractAt(ERC20_ABI, EIGEN, await ethers.getSigner(ethereum.NORMAL_TIMELOCK));
+    usdtPrimeConverter = await ethers.getContractAt(PRIME_CONVERTER_ABI, USDT_PRIME_CONVERTER);
+    usdt = await ethers.getContractAt(ERC20_ABI, BaseAssets[0], await ethers.provider.getSigner(USDT_USER));
+  });
+
+  testForkedNetworkVipCommands("vip393", await vip393(ONE_YEAR));
+
+  describe("Post-VIP behavior", async () => {
+    it("check price", async () => {
+      expect(await resilientOracle.getPrice(EIGEN)).to.be.equal(parseUnits("2.94577903", 18));
+      expect(await resilientOracle.getUnderlyingPrice(vEIGEN)).to.be.equal(parseUnits("2.94577903", 18));
+    });
+
+    it("should have 10 markets in core pool", async () => {
+      const poolVTokens = await comptroller.getAllMarkets();
+      expect(poolVTokens).to.have.lengthOf(10);
+    });
+
+    it("should add vEIGEN to the pool", async () => {
+      const registeredVToken = await poolRegistry.getVTokenForAsset(comptroller.address, EIGEN);
+      expect(registeredVToken).to.equal(vEIGEN);
+    });
+
+    it("check ownership", async () => {
+      expect(await vEIGENContract.owner()).to.equal(ethereum.GUARDIAN);
+    });
+
+    it("check supply", async () => {
+      const expectedSupply = parseUnits("349.618192", 8);
+      expect(await vEIGENContract.balanceOf(ethereum.VTREASURY)).to.equal(expectedSupply);
+    });
+
+    it("check borrow and supply caps", async () => {
+      expect(await comptroller.borrowCaps(vEIGEN)).equals(BORROW_CAP);
+      expect(await comptroller.supplyCaps(vEIGEN)).equals(SUPPLY_CAP);
+    });
+
+    it("should set vEIGEN collateral factor to 50% and Liquidation threshold to 60%", async () => {
+      const market = await comptroller.markets(vEIGEN);
+      expect(market.collateralFactorMantissa).to.equal(parseUnits("0.5", 18));
+      expect(market.liquidationThresholdMantissa).to.equal(parseUnits("0.6", 18));
+    });
+
+    it("check protocol share reserve", async () => {
+      expect(await vEIGENContract.protocolShareReserve()).equals(PROTOCOL_SHARE_RESERVE);
+    });
+
+    it("check reserve factor", async () => {
+      expect(await vEIGENContract.reserveFactorMantissa()).equals(parseUnits("0.25", 18));
+    });
+
+    it("check protocol seize share", async () => {
+      expect(await vEIGENContract.protocolSeizeShareMantissa()).equals(parseUnits("0.05", 18));
```

Comment:
Shouldn't protocol seize share be 1e16?

Reply:
The "protocol seize share" must be 5e16, as it is in the rest of the markets in the Core pool. The spect was wrong